bundle/jsky.app.ot/src/main/java/jsky/app/ot/editor/seq/DynamicSequenceTableModel.java

```diff
@@ -143,7 +139,7 @@ public void setSequence(ConfigSequence sequence, SPNodeKey key, List<ItemKey> al
             // Add it to the list for that parent.
             List<ItemKey> lst = groupedItems.get(parent.getName());
```

Comment:
I know this is nitpicky, but `final` could be used here with a second list, easily.
`final List<ItemKey> lst = groupedItems.get(parent.getName());`
`final List<ItemKey> finalLst;`
`if (lst == null) { finalList = new ArrayList<>(); groupedItems.put(parent.getName(), finalList); }`
`else finalList = lst;`